tests/InputFiles/ReduceCceWorldtube/ReduceCceWorldtube.yaml

```diff
@@ -0,0 +1,12 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+# [reduce_cce_worldtube_yaml_doxygen_example]
+
+InputSpecH5File: SpecFilenameR0292.h5
```

Comment:
`Spec` -> `Input`? Unless we actually have this file in the repo for testing, then leave as-is